scripts/twine_upload.sh

```diff
@@ -41,5 +41,7 @@ done
 
 # Upload the distributions.
 for p in dist/* ; do
-  twine upload $p
+  twine upload --verbose $p
```

Comment:
I think we should use [`--skip-existing`](https://github.com/pypa/twine/issues/115).

Reply:
Oh that's much better, thanks.